src/main/java/io/github/mountainrange/mule/controllers/PlayController.java

```diff
@@ -27,7 +26,7 @@
 
 	private SceneLoader sceneLoader;
 	private MULE mule;
-	private VisualGrid<VisualTile> g;
+	private VisualGrid g;
 	private WorldMap map;
```

Comment:
I'm not sure whether to make this a `WorldMap<Tile>` or `WorldMap<VisualTile>`; the compiler hates them both. I tried changing the GameManager constructor, but I had issues there as well. I think we would need to make GameManager generic for Tile as well.